tests/functional/aws-node-sdk/test/bucket/testBucketVersioning.js

```diff
@@ -146,11 +145,40 @@ describe('aws-node-sdk test bucket versioning', function testSuite() {
 
     it('should retrieve the valid versioning configuration', done => {
         const params = { Bucket: bucket };
-        // s3.getBucketVersioning(params, done);
         s3.getBucketVersioning(params, (error, data) => {
             assert.strictEqual(error, null);
             assert.deepStrictEqual(data, { Status: 'Enabled' });
             done();
         });
     });
 });
+
+describe('aws-node-sdk test bucket versioning with object lock', () => {
+    let s3;
+
+    // setup test
+    before(done => {
+        const config = getConfig('default', { signatureVersion: 'v4' });
+        s3 = new S3(config);
+        s3.createBucket({
+            Bucket: bucket,
+            ObjectLockEnabledForBucket: true,
+        }, done);
+    });
+
+    // delete bucket after testing
+    after(done => s3.deleteBucket({ Bucket: bucket }, done));
+
+    it('should not accept empty versioning configuration', done => {
```

Comment:
the test description might be better as "should not accept versioning configurations"

Reply:
looks like I forgot the description from above test, just updated.